x-pack/plugins/apm/public/components/routing/templates/settings_template.tsx

```diff
@@ -55,13 +62,20 @@ function getTabs({
   core,
   selectedTab,
   router,
+  defaultEnvironment,
 }: {
   core: CoreStart;
   selectedTab: Tab['key'];
   router: ApmRouter;
+  defaultEnvironment: Environment;
 }) {
   const canReadMlJobs = !!core.application.capabilities.ml?.canGetJobs;
 
+  const agentExplorerEnabled = core.uiSettings.get<boolean>(
```

Comment:
Yes, it's the only usage. Since this one is checking whether the `Agent Explorer` feature is enabled I don't think we will be reusing it in other places.